target/linux/bcm27xx/patches-5.15/950-0198-drm-fourcc-Add-packed-10bit-YUV-4-2-0-format.patch

```diff
@@ -14,21 +14,9 @@ Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
  include/uapi/drm/drm_fourcc.h | 11 +++++++++++
  2 files changed, 14 insertions(+)
 
---- a/drivers/gpu/drm/drm_fourcc.c
-+++ b/drivers/gpu/drm/drm_fourcc.c
-@@ -266,6 +266,9 @@ const struct drm_format_info *__drm_form
- 		  .num_planes = 3, .char_per_block = { 2, 2, 2 },
- 		  .block_w = { 1, 1, 1 }, .block_h = { 1, 1, 1 }, .hsub = 0,
- 		  .vsub = 0, .is_yuv = true },
-+		{ .format = DRM_FORMAT_P030,            .depth = 0,  .num_planes = 2,
-+		  .char_per_block = { 4, 4, 0 }, .block_w = { 3, 0, 0 }, .block_h = { 1, 0, 0 },
-+		  .hsub = 2, .vsub = 2, .is_yuv = true},
```

Comment:
I think this complete patch was backported, see https://github.com/gregkh/linux/commit/ec1727f89ecd6f2252c0c75e200058819f7ce47a

Reply:
Removed completely in https://github.com/openwrt/openwrt/commit/9b7d9a03a5872c39d1941f63872cbfeca9c9189d